src/test/regress/expected/with_transactions.out

```diff
@@ -105,6 +105,23 @@ SELECT count(*) FROM second_raw_table;
      0
 (1 row)
 
+-- sequential insert followed by a sequential real-time query should be fine
```

Comment:
Interesting test. Not for this PR, but now I kind of wonder whether we could make this automatic :)

Reply:
I'll keep an eye on this once we introduce automated switches.